### PR TITLE
fix: nanak coin on dashboard

### PR DESCRIPTION
**Pull request by amitojsingh**
1. Fix Nanak Coin issue

### CODE REVIEW

File: src/components/header/Header.tsx [hunk 1]
```diff
@@ -15,13 +16,21 @@ interface PropTypes {
 export default function Header({ ...props }: PropTypes) {
   const { t: text } = useTranslation();
   const { user } = useUserAuth();
-  const [ photoURL, setPhotoURL ] = React.useState('/images/profile.jpeg');
+  const [photoURL, setPhotoURL] = React.useState('/images/profile.jpeg');
```
Comment (MySmilingTurban) [inline src/components/header/Header.tsx:19]: remove the React as useState is already imported

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
minor change

File: src/components/header/Header.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/components/header/Header.tsx]: done veer ji

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**